src/frontend/src/components/Maps.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+
+import { getBackend } from '../lib/Backend';
+
+import {
+  Row,
+  Col,
+} from 'react-bootstrap';
+
+type Props = {};
+type State = {
+  maps: Array
+};
+
+class Maps extends Component<Props, State> {
+  constructor(props, state) {
+    super(props, state);
+    this.state = {
+      maps: [],
+    }
+    this.backend = getBackend().admin
+  }
+
+
+
+  componentDidMount() {
+    this.updateMaps()
+  }
+
+  updateMaps = async () => {
+    let maps = await this.backend.getMaps()
+    this.setState({ maps })
+  }
+
+  render() {
+    const { maps } = this.state;
+    return <Row className="mt-5 mb-5">
+      <Col lg={12} md={12} xs={12}>
+        Maps
+      </Col>
+      {
+        maps.map((map) => <Col key={map.id} lg={12} md={12} xs={12}>
+          {map.name}
+          <img src={`/maps/${map.filename}`} alt={map.filename} style={{ display: 'inline-block', height: '300px', width: 'auto' }} />

```

Comment:
@klemenkenda Agreed, there should be (almost) no inline css, unless needed (dynamic values). This will be updated in the UI step, the current version is just a placeholder.

Do you want to use .css files, styled components or something else? If we will use css directly we should implement a consistent naming scheme e.g. http://getbem.com/naming/. Otherwise [styled components](https://github.com/styled-components/styled-components) are probably the best solution.

Reply:
BEM looks a bit ugly, but - yes, why not go with it. I know no alternative.
I wouldn't use styled components as they mix CSS with code.